src/k8s-engine/lima.ts

```diff
@@ -1812,6 +1815,40 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     await this.ssh('sudo', 'update-ca-certificates');
   }
 
+  protected async getHostIPAddr(): Promise<string> {
+    try {
+      const lines = (await this.limaWithCapture('shell', '--workdir=.', MACHINE_NAME, 'ip', 'route', 'list', 'eth0')).split(/\n/);
+      const fields = lines[0].split(/\s+/);
+
+      return fields[2];
+    } catch (err: any) {
+      console.log(`ip route failed: ${ err }`, err);
+      throw err;
+    }
+  }
+
+  protected async installCredentialHelper() {
+    const credsPath = getServerCredentialsPath();
+
+    try {
+      const hostIPAddr = await this.getHostIPAddr();
+      const stateInfo: ServerState = JSON.parse(await fs.promises.readFile(credsPath, { encoding: 'utf-8' }));
```

Comment:
Hmm, so this is safe because we launch the cred server super early, so it would have written out the info already.
(It seems odd that we're going via the file on disk instead of going in-process, but 🤷)

Reply:
I didn't want to couple the Credential Server (nor the Command Server) with the backend, so it isn't given the data.